client/api_group.go

```diff
@@ -32,7 +33,7 @@ type Group interface {
 	// addAddresses indicates the HEX-encoded string list of the member addresses to be added
 	// removeAddresses indicates the HEX-encoded string list of the member addresses to be removed
 	UpdateGroupMember(ctx context.Context, groupName string, groupOwnerAddr string,
```

Comment:
add a comment to explain expiration time . For example "	// expirationTime defines the expiration time of the group member， user need set the expiration time for the addAddresses "